### PR TITLE
feat: new UI features for the package list

### PR DESCRIPTION
**Pull request by spalt08**

### CODE REVIEW

File: packages/web/src/components/ui/Package/Package.tsx [hunk 1]
```diff
@@ -19,19 +27,82 @@ export type Props = {
...
+  let byteUnitIndex = 0;
+  let sizeInUnits = size;
+
+  while (sizeInUnits > 1024) {
+    sizeInUnits /= 1024;
+    byteUnitIndex += 1;
+  }
+
+  let readableSize = sizeInUnits.toFixed(1);
+  if (readableSize.slice(-2) === '.0') {
```
Comment (ctizen) [inline packages/web/src/components/ui/Package/Package.tsx:95]: `.endsWith()`?

File: packages/web/src/components/ui/Package/Package.tsx [hunk 2]
```diff
@@ -19,19 +27,82 @@ export type Props = {
...
+          {repositoryUrl && (
+            <a
+              href={repositoryUrl}
+              target='_blank'
+              rel='noopener noreferrer'
+              className={styles.externalLink}
+            >
+              <Github />
+            </a>
+          )}{' '}
```
Comment (ctizen) [inline packages/web/src/components/ui/Package/Package.tsx:49]: This space will appear always, which is probably not the desired behavior. This should be done with css, I guess.

File: packages/web/src/components/ui/Package/Package.tsx (hunk 1 above)
Comment (zardak) [reply packages/web/src/components/ui/Package/Package.tsx:95]: There's also a more common but less readable approach of `parseFloat(float.toFixed(x))` which does the trick

**Review by zardak — APPROVED** [review]

File: packages/web/src/components/ui/Package/Package.tsx (hunk 1 above)
Comment (spalt08) [reply packages/web/src/components/ui/Package/Package.tsx]: Fixed

File: packages/web/src/components/ui/Package/Package.tsx (hunk 2 above)
Comment (spalt08) [reply packages/web/src/components/ui/Package/Package.tsx]: Fixed

**Review by ctizen — APPROVED** [review]

**Pull request merged**